packages/react/src/Card/CardHeading.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { CardHeading } from './CardHeading'
+import '@testing-library/jest-dom'
+
+describe('Card Heading', () => {
+  it('renders', () => {
+    const { container } = render(<CardHeading />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<CardHeading />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-card__heading')
+  })
+
+  it('renders an additional class name', () => {
+    const { container } = render(<CardHeading className="extra" />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-card__heading extra')
+  })
+
```

Comment:
We're missing a test for the `size` prop here. And `level`, if we keep it.

Reply:
Card Heading outsources its rendering to Heading, and Heading tests its size prop. That seems enough, right?

Reply:
Not really. `CardHeading` has a `size` prop, which has it's own logic (albeit very simple logic 😁). That logic isn't tested now. If we mess up passing the value of `size` to the Heading in the future, our tests would not tell us.

Reply:
Yes, you’re right. Added it.